src/components/Teams.js

```diff
@@ -0,0 +1,189 @@
+import styled from 'styled-components'
+import { Column, Container, Row, GradientText, Circle } from './common'
+import Image from 'next/image'
+
+const YourTeams = styled.div`
+  font-weight: 600;
+  font-size: 26px;
+  line-height: 31px;
+  line-height: 100%;
+  transform: translateY(-86px);
+  `
+
+
+const Box = styled(Container)`
+  height: 512px;
+  border-radius: 5px;
+  margin-top: 86px;
+  background: ${props => props.theme.colors.white};
+  position: relative;
+`
+
+
+
+const TeamName = styled.div`
+  font-family: Inter;
+  font-weight: 600;
+  font-size: 40px;
+  line-height: 48px;
+  line-height: 100%;
+`
+
+const GreyText = styled.div`
+  font-family: Inter;
+  font-style: Regular;
+  font-size: 16px;
+  fine-height: 24px;
+  color: ${props => props.theme.colors.gray};
+  letter-spacing: 0.1em;
+`
+
+
+const Numbers = styled.div`
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  margin-top: 6px;
+`
+const Laptop = styled(Circle)`
+  width: 39.08px;
+  height: 39.08px;
+  margin-right: 13.98px;
+  box-shadow: 0px 4px 8px 0px #0000000D;
+`
+
+const Xbox= styled(Circle)`
+  width: 39.08px;
+  height: 39.08px;
+  margin-right: 44.89px;
+  box-shadow: 0px 4px 8px 0px #0000000D;
+`
+
+const Spacer = styled(Row)`
+  justify-content: space-between;
+`
+
+const Delete = styled(Column)`
+  width: 674px;
+  height: 205px;
+  background: red;
+  transform: translateY(154px);
+  z-index: 30;
+`
+
+export default function Teams() {
+
+  const teammates = [
+    {
+      user: 'devonhenry_'
+    },
+    {
+      user: 'devonhenry_'
+    },
+    {
+      user: 'devonhenry_'
+    }
+  ]
+
+  const teams = [
+    {
+      name: '[INSERT TEAM NAME]',
+      wins: 123,
+      losses: 4,
+      prem: 1234567
+    },
+    {
+      name: '[INSERT TEAM NAME]',
+      wins: 123,
+      losses: 4,
+      prem: 1234567
+    },
+    {
+      name: '[INSERT TEAM NAME]',
+      wins: 123,
+      losses: 4,
+      prem: 1234567
+    }
+  ]
+
+  return (
+      <div>
+        {teams.map((team) => 
+          <div>
+          <Box style ={{ marginBottom: 50 }}>
+            <Column>
+              <Spacer>
+                <Row style={{ marginTop: 63, marginLeft: 74, }}>
```

Comment:
it is a good practice to make 
```js
const TeammatesRow = styled(Row)``
```
```jsx
<TeammatesRow>
  ...
</TeammatesRow>
```
and always make another styled component using the base one, as later on it makes mobile scaling more convenient and makes the code more readable

Reply:
do the same thing for any ```<div>``` that you put in the component body, except for when you need to add a tiny bit of margin to an image